src/services/api.ts

```diff
@@ -350,10 +350,14 @@ export class YdbEmbeddedAPI extends AxiosWrapper {
             enable_sampling: enableSampling,
         });
     }
-    getHealthcheckInfo(database: string) {
-        return this.get<HealthCheckAPIResponse>(this.getPath('/viewer/json/healthcheck'), {
-            tenant: database,
-        });
+    getHealthcheckInfo(database: string, {concurrentId}: AxiosOptions = {}) {
+        return this.get<HealthCheckAPIResponse>(
+            this.getPath('/viewer/json/healthcheck'),
+            {
+                tenant: database,
+            },
+            {concurrentId: concurrentId || `getHealthcheckInfo|${database}`},
```

Comment:
Default name isn't correct here as it doesn't fix bug with blinking. In this case it's better not to pass default name at all and left `{tenant: database}, {concurrentId}`